sql/service/src/main/java/org/apache/spark/sql/service/cli/Type.java

```diff
@@ -22,62 +22,66 @@
 import java.util.Locale;
 
 import org.apache.spark.sql.service.rpc.thrift.TTypeId;
-import org.apache.spark.sql.types.DecimalType;
+import org.apache.spark.sql.types.*;
 
 /**
  * Type.
  *
  */
 public enum Type {
-  NULL_TYPE("VOID",
+  NULL_TYPE(NullType$.MODULE$.sql(),
           java.sql.Types.NULL,
           TTypeId.NULL_TYPE),
-  BOOLEAN_TYPE("BOOLEAN",
+  BOOLEAN_TYPE(BooleanType$.MODULE$.sql(),
           java.sql.Types.BOOLEAN,
           TTypeId.BOOLEAN_TYPE),
-  TINYINT_TYPE("TINYINT",
+  TINYINT_TYPE(ByteType$.MODULE$.sql(),
           java.sql.Types.TINYINT,
           TTypeId.TINYINT_TYPE),
-  SMALLINT_TYPE("SMALLINT",
+  SMALLINT_TYPE(ShortType$.MODULE$.sql(),
           java.sql.Types.SMALLINT,
           TTypeId.SMALLINT_TYPE),
-  INT_TYPE("INT",
+  INT_TYPE(IntegerType$.MODULE$.sql(),
           java.sql.Types.INTEGER,
           TTypeId.INT_TYPE),
-  BIGINT_TYPE("BIGINT",
+  BIGINT_TYPE(LongType$.MODULE$.sql(),
           java.sql.Types.BIGINT,
           TTypeId.BIGINT_TYPE),
-  FLOAT_TYPE("FLOAT",
+  FLOAT_TYPE(FloatType$.MODULE$.sql(),
           java.sql.Types.FLOAT,
           TTypeId.FLOAT_TYPE),
-  DOUBLE_TYPE("DOUBLE",
+  DOUBLE_TYPE(DoubleType$.MODULE$.sql(),
           java.sql.Types.DOUBLE,
           TTypeId.DOUBLE_TYPE),
-  STRING_TYPE("STRING",
+  STRING_TYPE(StringType$.MODULE$.sql(),
           java.sql.Types.VARCHAR,
           TTypeId.STRING_TYPE),
-  DATE_TYPE("DATE",
+  DATE_TYPE(DateType$.MODULE$.sql(),
           java.sql.Types.DATE,
           TTypeId.DATE_TYPE),
-  TIMESTAMP_TYPE("TIMESTAMP",
+  TIMESTAMP_TYPE(TimestampType$.MODULE$.sql(),
           java.sql.Types.TIMESTAMP,
           TTypeId.TIMESTAMP_TYPE),
-  BINARY_TYPE("BINARY",
+  // TODO: Map INTERVAL_TYPE to CalendarIntervalType
+  INTERVAL_TYPE(CalendarIntervalType$.MODULE$.sql(),
```

Comment:
To actually add INTERVAL_TYPE, we would need to extend the wire protocol... I would not do that now just for the sake of adding INTERVAL_TYPE.

Reply:
OK. will remove INTERVAL_TYPE